### PR TITLE
feat: add kafka tls flags

### PR DESCRIPTION
**Pull request by dennisme**
Adding required flags

### CODE REVIEW

File: protos/args/ps_args_kafka.proto [hunk 1]
```diff
@@ -23,6 +23,15 @@ message KafkaConn {
   // @gotags: kong:"help='Allow insecure TLS usage',env=PLUMBER_RELAY_KAFKA_INSECURE_TLS"
   bool tls_skip_verify = 4;
 
+  // @gotags: kong:"help='TLS ca cert',env=PLUMBER_RELAY_TLS_CA_CERT"
+  bool tls_ca_cert = 3;
```
Comment (blinktag) [inline protos/args/ps_args_kafka.proto:27]:
```suggestion
  bool tls_ca_cert = 8;
```

File: protos/args/ps_args_kafka.proto [hunk 2]
```diff
@@ -23,6 +23,15 @@ message KafkaConn {
...
+
+  // @gotags: kong:"help='TLS client cert',env=PLUMBER_RELAY_TLS_CLIENT_CERT"
+  bool tls_client_cert = 3;
```
Comment (blinktag) [inline protos/args/ps_args_kafka.proto:30]:
```suggestion
  bool tls_client_cert = 9;
```

File: protos/args/ps_args_kafka.proto [hunk 3]
```diff
@@ -23,6 +23,15 @@ message KafkaConn {
...
+
+  // @gotags: kong:"help='TLS client key',env=PLUMBER_RELAY_TLS_CLIENT_KEY"
+  bool tls_client_key = 3;
```
Comment (blinktag) [inline protos/args/ps_args_kafka.proto:33]:
```suggestion
  bool tls_client_key = 10;
```

**Review by blinktag — CHANGES_REQUESTED** [review]
Field IDs need to be unique in the message. The highest currently is sasl_password, so start after that and increment

File: protos/args/ps_args_kafka.proto (hunk 1 above)
Comment (dennisme) [reply protos/args/ps_args_kafka.proto:27]: oh yes

**Comment by dennisme** [comment]
Believe this is failing because its on a fork.

**Comment by dselans** [comment]
This is super old; closing.

**Pull request closed without merging**